File: CodeUp/2775.c
#include <stdio.h>

int main()
{
    double hp, shield, time, dmg, ignore;

    scanf("%lf %lf %lf", &hp, &shield, &time);
    scanf("%lf", &dmg);   

    ignore = ((1 - (100 - (hp/time/dmg * 100)) / shield) * 100);

    if(ignore < 0)
        printf("O\n");
    else if(ignore > 100)
        printf("X\n");
    else
        printf("%.6lf\n", ignore);
}

File: codeup/2775/2775.py
hp, shield, time = map(float, input().split(' '))
dmg = float(input())

ignore = ((1 - (100 - (hp/time/dmg * 100)) / shield) * 100)

if ignore < 0:
    print("O")
elif ignore > 100:
    print("X")
else:
    print("%.6f" % ignore)